src/onboarding/welcome/Welcome.tsx

```diff
@@ -26,6 +31,14 @@ export default function Welcome() {
   const startOnboardingTime = useSelector((state) => state.account.startOnboardingTime)
   const insets = useSafeAreaInsets()
   const recoveringFromStoreWipe = useSelector(recoveringFromStoreWipeSelector)
+  const [termsCheckbox, toggleTermsCheckBox] = useState(acceptedTerms)
+
+  const { variant } = getExperimentParams(
+    ExperimentConfigs[StatsigExperiments.ONBOARDING_TERMS_AND_CONDITIONS]
+  )
+
+  const showTermsCheckbox = variant === 'checkbox'
```

Comment:
nit: 'control' | 'colloquial_terms' | 'checkbox' could be an enum in statsig/constants.ts

Reply:
we get type safety from the string constant types, and given that this is relatively short lived, don't want to add another enum for this.